docs/usb/wdi-usb-interface.md

```diff
@@ -33,13 +38,29 @@ https://gitlab.freedesktop.org/libevdev/libevdev/-/blob/master/include/linux/lin
  BTN_THUMBR
 ```
 
-## Drive actions
+## Drive Actions
 | Action | Keyboard Controls | Gamepad Controls |
 |--------|------|-----|
 | Drive forward | 'w' \| up arrow | Y axis (towards min val) |
 | Drive backward | 's' \| down arrow | Y axis (towards max val) |
 | Drive left | 'a' \| left arrow | X axis (towards min val) |
 | Drive right | 'd' \| right arrow | X axis (towards max val) |
-| Emergency stop | backspace | button_east?? would like this to be something that is easy to press under stress |
-| Enable drive | enter | start button |
-| 
\ No newline at end of file
+| Emergency stop | backspace | BTN_SOUTH \| BTN_THUMBL?? |
+| Enable device control | enter | BTN_START |
```

Comment:
I feel like enabling the device is more a LUCI implementation need. I'm assuming if this were integrated into the wheelchair electronics this would just happen when the chair is turned on. So I'm wondering if/where this should be included.